src/compute/divide.ts

```diff
@@ -0,0 +1,42 @@
+import { Image } from '../Image';
+import { getOutputImage } from '../utils/getOutputImage';
+import { validateChannels } from '../utils/validators/validators';
+
+//The used options are the same as in multiply function.
+import { MultiplyOptions } from './multiply';
+
+/**
+ *
+ * Divides image pixels by a certain value.
+ * @param image - image to which division will be applied.
+ * @param value - Value by which each pixel will be divided.
+ * @param options - Multiply options
+ * @returns image.
+ */
+export function divide(
+  image: Image,
+  value: number,
+  options: MultiplyOptions = {},
+) {
+  const {
+    channels = new Array(image.components).fill(0).map((value, index) => index),
+  } = options;
+  validateChannels(channels, image);
+  if (value === 0) {
+    throw new Error(`Value cannot be equal to 0.`);
```

Comment:
```suggestion
    throw new TypeError(`Cannot divide by 0`);
```

1. Use TypeError when you receive a bad argument
2. We don't put periods in our error messages.
3. Try to write the message in a way that the error is easy to interpret by the user.